cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -619,10 +619,20 @@ For services, you can retrieve:
 
 For workflows, you can retrieve:
 
-- the information about the specific run currently running::
```

Comment:
No in fact the output is a ``List<WorkflowActionNode>``, as json. So this should be ``node(s)``